tests/e2e.py

```diff
@@ -94,7 +94,7 @@ def handle(route: Route):
     # Check initial page state
     page.goto(live_server_url)
     expect(page).to_have_title("AI Document Explorer")
-    expect(page.get_by_role("heading", name="Chat with your data")).to_be_visible()
+    expect(page.get_by_role("heading", name="Ask PIM", exact=True)).to_be_visible()
```

Comment:
what does exact=True do?

Reply:
matches the exact text if multiple heading are returned, since we now have multiple :)